yarn.lock

```diff
@@ -1974,7 +1974,7 @@
     write-file-atomic "^2.3.0"
 
 "@microsoft/teams-js@file:packages/teams-js":
-  version "2.0.0-beta.0"
+  version "2.0.0-beta.1"
```

Comment:
This is interesting. Should we be changing the package.json to be beta.2 and targeting that instead? Or is it as designed to trail one beta version behind? (I feel like this is also more of a question for folks who's been here longer)

Reply:
I'll be bumping the version in a separate change for this next release.